.template-helpers/variant.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+repo="$1"
+if [ -z "$repo" ]; then
+	echo >&2 "usage: $0 repo"
+	echo >&2 "   ie: $0 hylang"
+	exit 1
+fi
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+IFS=$'\n'
+lines=( $(curl -sSL 'https://raw.githubusercontent.com/docker-library/official-images/master/library/'"$repo" | grep -vE '^$|^#') )
```

Comment:
Technically, we could just directly `... | grep ... | cut -d':' -f1 | sort -u` here, right?